src/test/java/com/snowflake/kafka/connector/UtilsTest.java

```diff
@@ -133,4 +136,23 @@ public void testSetJDBCLoggingDir() {
     Utils.setJDBCLoggingDirectory();
     assert System.getProperty(Utils.JAVA_IO_TMPDIR).equals(defaultTmpDir);
   }
+
+  @Test
+  public void testCollectSchemaFromTopics() throws Exception {
+    SchemaRegistryClient schemaRegistry = new MockSchemaRegistryClient();
+    schemaRegistry.register(
+        "topic0-value", new AvroSchema(TestUtils.AVRO_SCHEMA_FOR_SCHEMA_COLLECTION_0));
+    schemaRegistry.register(
+        "topic1-value", new AvroSchema(TestUtils.AVRO_SCHEMA_FOR_SCHEMA_COLLECTION_1));
+    Map<String, String> topicToTableMap = new HashMap<>();
+    topicToTableMap.put("topic0", "table");
+    topicToTableMap.put("topic1", "table");
```

Comment:
if topic0 table1 and topic1 and table2 are mapped, we will not create the schema map right?

Reply:
if we query for table1 or table2 we will still get the schema map